ElastosBountyProgram/front-end/src/module/page/developer/Component.js

```diff
@@ -165,27 +166,28 @@ export default class extends StandardPage {
         )
     }
 
+    getAvatarWithFallback(avatar) {
+        return _.isEmpty(avatar)
+            ? USER_AVATAR_DEFAULT
+            : avatar
+    }
+
     buildMemberSearch() {
-        const columns = [
-            {
-                title: 'Name',
-                dataIndex: 'name',
-                width: '33%',
-                render: (name, user) => this.getUserNameWithFallback(user)
-            },
-            {
-                title: 'Username',
```

Comment:
I like that you added avatar thats perfect - but id keep the username and circles columns